src/store/CrosshairStore.ts

```diff
@@ -236,7 +236,7 @@ class CrosshairStore {
                 dupMap.Open = dupMap.High = dupMap.Low = 1;
             }
         }
-        if (this.showSeries) {
+        if (this.showSeries && this.mainStore.timeperiod.timeUnit === 'tick') {
```

Comment:
`this.showSeries` is equal to `true`, so I tried to add `this.mainStore.timeperiod.timeUnit === 'tick'` instead of default value, but failed as `mainStore` is not initialised there. I'm thinking of removing `this.showSeries` at all from this file as it's used only in this line. Please, share your thoughts, as it's scary to delete smth from SmartCgart)
<img width="665" alt="Screenshot 2023-09-11 at 1 11 02 PM" src="https://github.com/binary-com/SmartCharts/assets/121025168/db881eae-265d-47a0-96e8-14609bc9452b">

Reply:
@kate-deriv please elaborate on how this change is related to the date format in the price info box? :) If we need to only change the format of the date for all possible cases, we don't need to change this line, do we?

Reply:
@maryia-deriv It is not related to the date format, it is related to removing symbol field in chart info-box for non-ticks time intervals (please, check the notes in the card, this PR is common for 2 cards)